index.html

```diff
@@ -28,10 +28,122 @@ <h1>Felipe<br>Oyarzo</h1>
                     my works. If you like what you see and have a project you need coded, don't hesitate and contact me.
                 </p>
             </div>
-            <button class="button" type="button" id="btnHeadline">Start collaboration</button>
-            <div class="geometric"></div>            
+            <button class="button" type="button" id="btnHeadline">Start collaboration</button>            
         </section>
     </main>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you add the missing background image to make your design sticks to your chosen Figma template.

| Figma DEsign | Yor Design |
| -------------------- | --------------------- |
| ![Screenshot from 2022-08-20 20-17-26](https://user-images.githubusercontent.com/83042742/185761131-1274f616-a5d5-4ea8-ab89-3eb3e38ff6d6.png) | ![Screenshot from 2022-08-20 20-16-57](https://user-images.githubusercontent.com/83042742/185761144-77175348-b8a9-41cb-8e95-fb9ab679266f.png) |

Reply:
Hello @tresorsawasawa , thank you very much for your feedback. It's strange because in my localhost I can see the background images without a problem. What could be the issue? Are there any more background images that are not showing on your side? thank you kindly for your help
![image](https://user-images.githubusercontent.com/101157253/185761959-672c5d17-ab6a-437b-aa0f-e9680e12462a.png)